spec/spec_helper.rb

```diff
@@ -1,6 +1,9 @@
+require 'simplecov'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.